tensorflow/lite/g3doc/inference_with_metadata/task_library/image_classifier.md

```diff
@@ -99,6 +99,140 @@ See the
 [source code and javadoc](https://github.com/tensorflow/tflite-support/blob/master/tensorflow_lite_support/java/src/java/org/tensorflow/lite/task/vision/classifier/ImageClassifier.java)
 for more options to configure `ImageClassifier`.
 
+## Run inference in iOS
+
+### Step 1: Install the dependencies
+
+The task library supports installation using cocoapods. Make sure that cocoapods is installed on your system. 
+Please see the [cocoapods installation guide](https://guides.cocoapods.org/using/getting-started.html#getting-started) for instructions.
+
+Please see the [cocoapods guide](https://guides.cocoapods.org/using/using-cocoapods.html) for details on adding pods to an Xcode project. 
+
+Add the TensorFlowLiteTaskVision pod in Podfile
+
+```
+target 'MyAppWithTaskAPI' do
+  use_frameworks!
+  pod 'TensorFlowLiteTaskVision', '~>0.0.1-nightly.20220508'
+end
+```
+
+Make sure the `.tflite` you will be using for inference is present in your app bundle.
+
+### Step 2: Using the model 
+
+#### Swift
+
+```swift
+// Imports
```

Comment:
I'd suggest reformat the code like this.
The do / catch makes the code a bit hard to read. Keyword `try` present there should be enough to let Swift users know that there's error thrown and they need to handle it.

```swift
// Initialization
guard let modelPath = Bundle.main.path(forResource: "mobilenetv3",
                                            ofType: "tflite") else { return }
let options = ImageClassifierOptions(modelPath: modelPath)

// Optional: Configure any additional options:
// options.classificationOptions.maxResults = 5

let classifier = try ImageClassifier.imageClassifier(options: options)

// Convert the input image to MLImage.
// There are other sources for MLImage. For more details, please see:
// https://developers.google.com/ml-kit/reference/ios/mlimage/api/reference/Classes/GMLImage
guard let image = UIImage (named: "burger.jpg"), let mlImage = MLImage(image: image) else { return }

// Run inference
let classificationResults = try classifier.classify(gmlImage: mlImage)
```

Reply:
Changed the model name and image name based on the "Example Results" section for each task. For eg., image classifier illustrates results for sparrow.jpg on a bird classifier.